src/Common/Config/YAMLParser.cpp

```diff
@@ -136,6 +136,80 @@ void processNode(const YAML::Node & node, Poco::XML::Element & parent_xml_elemen
     }
 }
 
+/// Daisy : starts
+bool processNodeToJson(const YAML::Node & ynode, Poco::Dynamic::Var & jnode)
```

Comment:
Let's move the 2 new functions to our Rest folder to a new file. Rename `processNodeToJson` to `yamlNodeToJSONNode`

Reply:
fixed